SimpleSvm/SimpleSvm.cpp

```diff
@@ -637,7 +637,10 @@ SvHandleMsrAccess (
         //
         // This code does not implement the check intentionally, for simplicity.
         //
-        VpData->GuestVmcb.StateSaveArea.Efer = value.QuadPart;
+        else
+        {
+            VpData->GuestVmcb.StateSaveArea.Efer = value.QuadPart;
+        }
```

Comment:
we should be `return`-ing above. Otherwise, it advances guest RIP below, which is incorrect

Reply:
Yeah, youre right. I'm not too good with git, is it possible for you to propose the change and pull?

Reply:
you can make the changes in your `patch-1` branch by simply adding a new commit and pushing as usual. GitHub automatically reflect the changes into this PR. 

Feel free to use this as a learning opportunity.